setup.py

```diff
@@ -48,6 +48,6 @@
       packages=['textfsm'],
       include_package_data=True,
       package_data={'textfsm': ['../testdata/*']},
-      install_requires=['six', 'future'],
+      install_requires=['six', 'future', 'regex'],
```

Comment:
I'm conflicted here ... we want to opportunistically install regex but not necessarily require it, as only newer templates would need it. Unfortunately I see no provision in setup.py for a install_desires=... stanza, so I guess this will have to do?

Reply:
Its the templates that drive the dependency. i.e. only if there are templates with the Repeated keyword, do we need the regexp module installed (plus the relevant unittests). It could be a declared as required as part of a template package install rather than here .. but that is perhaps asking to much of the template maintainers.